hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -2639,10 +2639,11 @@ HdfsFileStatus startFile(String src, PermissionStatus permissions,
           createParent, replication, blockSize, supportedVersions, ecPolicyName,
           storagePolicy, logRetryCache);
     } catch (AccessControlException e) {
-      logAuditEvent(false, "create", src);
+      logAuditEvent(false, "create(options=" + flag.toString() + ")", src);
```

Comment:
can we leave a space after "create" so it is consistent with rename? e.g., "create (options=..."

Reply:
thanks for your suggestion.
I will modify it.